### PR TITLE
Updating bootstrap

### PR DESCRIPTION
**Pull request by nmcspadden**

### CODE REVIEW

**Comment by cgerke** [comment]
Not sure if you want to generalise line 69...

File: chef/tools/chef_bootstrap.py [hunk 1]
```diff
@@ -285,50 +279,102 @@ def install_cli_tools():
...
+    if install_path.endswith('.dmg'):
+      # Search the DMG for a valid .pkg
+      pkg_path = get_pkg_path_from_dmg(install_path)
+      if not pkg_path:
+        # Can't find a package inside the DMG.
+        return False
+      # Copy the Chef installer to /Library/Chef/Source while we're at it
+      if not os.path.isdir('/Library/Chef/Source'):
+        os.makedirs('/Library/Chef/Source', mode=0755)
+      shutil.copy2(pkg_path, '/Library/Chef/Source/')
```
Comment (clburlison) [inline chef/tools/chef_bootstrap.py]: Need `import shutil`

File: chef/tools/chef_bootstrap.py [hunk 2]
```diff
@@ -417,7 +466,7 @@ def bootstrap(force=False):
     f.write(CLIENT_RB)
   with open('/etc/chef/run-list.json', 'wb') as f:
     json.dump(RUN_LIST_JSON, f)
-  with open('/etc/chef/org.crt', 'wb') as f:
+  with open('/etc/chef/thefacebook.crt', 'wb') as f:
```
Comment (clburlison) [inline chef/tools/chef_bootstrap.py]: Revert to `/etc/chef/org.crt` for generialization purposes

File: chef/tools/chef_bootstrap.py [hunk 3]
```diff
@@ -462,7 +511,56 @@ def bootstrap(force=False):
...
+  """Set the Computer, Host, LocalHost Names."""
+  scutil_cmd = ['/usr/sbin/scutil', '--set']
+  run_subp(scutil_cmd + ['ComputerName', name])
+  run_subp(scutil_cmd + ['HostName', name])
+  run_subp(scutil_cmd + ['LocalHostName', name])
+
+
 if __name__ == '__main__':
-  result = bootstrap(force=True)
+  parser = argparse.ArgumentParser(
```
Comment (clburlison) [inline chef/tools/chef_bootstrap.py]: Missed one, also needs `import argparse`

**Pull request closed without merging**